front-end/src/app/app.component.html

```diff
@@ -18,7 +18,7 @@
   </a>
 
   <div class="right menu">
-    <div class="item" *ngIf="connectionLost" suiPopup popupInverted="true" popupText="The server may be restarting or down for maintenance">
+    <div class="item" *ngIf="connectionLost" suiPopup popupInverted="true" popupText="The server may be restarting or down for maintenance. Check your internet, maybe?">
```

Comment:
'Check your internet' doesn't really apply here. It could be explained better but the `connectionLost` variable is set to true if the `GET` request returns a specific status code (504). A user will not (as far as I know) see this if their internet is down, but only if the server doesn't respond.

I think this might be some chrome magic, but whatever.

Reply:
I see. I haven't looked at that code yet, so I assumed it was something like pinging the API. I'll revert it.